Makefile

```diff
@@ -170,3 +172,21 @@ pao-functests-only:
 cluster-clean-pao:
 	@echo "Cleaning up performance addons artifacts"
 	hack/clean-deploy.sh
+
+# Performance Profile Creator (PPC)
+.PHONY: build-tools-bin-dir
+build-tools-bin-dir:
+	mkdir -p $(TOOLS_BIN_DIR)
+
+
+.PHONY: build-performance-profile-creator
+build-performance-profile-creator: | build-tools-bin-dir
+	@echo "Building Performance Profile Creator (PPC)"
+	LDFLAGS="-s -w "; \
+	LDFLAGS+="-X github.com/openshift-kni/cluster-node-tuning-operator/cmd/performance-profile-creator "; \
+	env GOOS=$(GOOS) GOARCH=$(GOARCH) $(GO) build  -v $(LDFLAGS) -o $(TOOLS_BIN_DIR)/performance-profile-creator ./cmd/performance-profile-creator
```

Comment:
we can perhaps take the chance to review and simplify this command

Reply:
maybe even reuse the variables at the top of the file

Reply:
I am not sure if the changes I have already done are the ones you were aiming for ...